polysh/control_commands.py

```diff
@@ -371,8 +337,8 @@ def do_set_log(command):
     command = command.strip()
     if command:
         try:
-            remote_dispatcher.options.log_file = file(command, 'a')
```

Comment:
I didn't know `file()` being old spelling of `open()`.  It is nice to finally figure out why my editor highlights variables named as `file`.

Reply:
Can I somehow mark this as resolved like in gitlab?